File: Baitap06/app/src/main/java/com/example/baitap06/MenuActivity.java
package com.example.baitap06;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.example.baitap06.databinding.ActivityMenuBinding;

public class MenuActivity extends AppCompatActivity {

    private ActivityMenuBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMenuBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.btnFragmentExample.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MenuActivity.this, MainActivity.class));
            }
        });

        binding.btnRecyclerExample.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MenuActivity.this, RecyclerViewActivity.class));
            }
        });

        binding.btnFlipperExample.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MenuActivity.this, FlipperActivity.class));
            }
        });

        binding.btnViewPagerExample.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MenuActivity.this, ViewPagerActivity.class));
            }
        });
    }
}

File: Baitap07/app/src/main/java/com/example/baitap07/MenuActivity.kt
package com.example.baitap07

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MenuActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_menu)

        val btnBai1: Button = findViewById(R.id.btnBai1)
        val btnBai2: Button = findViewById(R.id.btnBai2)

        // Mở Bài 1: Hồ sơ & Upload ảnh (ProfileActivity)
        btnBai1.setOnClickListener {
            val intent = Intent(this, ProfileActivity::class.java)
            startActivity(intent)
        }

        // Mở Bài 2: Điều khiển thiết bị (SocketActivity)
        btnBai2.setOnClickListener {
            val intent = Intent(this, SocketActivity::class.java)
            startActivity(intent)
        }
    }
}
